hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/pipeline/Pipeline.java

```diff
@@ -207,6 +222,8 @@ public static Pipeline getFromProtobuf(HddsProtos.Pipeline pipeline)
         .setFactor(pipeline.getFactor())
         .setType(pipeline.getType())
         .setState(PipelineState.fromProtobuf(pipeline.getState()))
+        .setLeaderId(StringUtils.isNotEmpty(pipeline.getLeaderID()) ?
```

Comment:
lets change this to pipeline.hasLeaderID().

Reply:
Does it cover for empty "" case?